src/main/java/uk/co/ramp/covid/simulation/lockdown/easingevents/CommunalPlaceEasingEvent.java

```diff
@@ -10,8 +10,8 @@
 
 /** Lockdown component to ease restrictions on places */
 public abstract class CommunalPlaceEasingEvent extends LockdownEvent {
-    private Probability keyPremises = null;
-    private Double socialDistance = null;
+    private final Probability keyPremises;
```

Comment:
Just as a note for reference (I'm assuming this works since the tests pass!) in case this bites us later. These were explicitly set to null since it wasn't clear how both "final" and no value worked when initialising with gson (in the case no value is provided, i.e. does it definitely default to null). Because these are set through reflection in gson this tends to confuse IntelliJ!

Reply:
I'm fairly sure that Java doesn't have uninitialized variables, and things will always be initialized with the default value (null in this case) when they're not initialized with something else.  So adding `= null` has no effect.